devtools/environment.yml

```diff
@@ -3,8 +3,8 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - MDAnalysis<2.0.0
-  - MDAnalysisTests<2.0.0
+  - MDAnalysis
```

Comment:
It'll at least need to be >1, but I agree with @orbeckst, we should just move to 2.0+ at this point.

Reply:
@IAlibay so `>2.0.0` is fine?

Reply:
Yup